README.tpl

```diff
@@ -1,42 +0,0 @@
-**_Please be aware that version `v0.4.0` of rusty-cheddar will be based around the more general
```

Comment:
Ah, _that's_ what it's for! Ok, that makes sense. But why on earth doesn't cargo-readme use the markdown version?